### PR TITLE
Add WordPress configurations for site publish.

### PR DESCRIPTION
**Pull request by mrrobot47**
Depends on: https://github.com/EasyEngine/site-command/pull/260
Issue ref: https://github.com/EasyEngine/feature-requests/issues/64

Signed-off-by: Riddhesh Sanghvi <riddheshsanghvi96@gmail.com>

### CODE REVIEW

File: src/WordPress.php [hunk 1]
```diff
@@ -1296,6 +1296,49 @@ public function reload( $args, $assoc_args, $whitelisted_containers = [], $reloa
...
+	 *
+	 * [--disable]
+	 * : Take online link down.
+	 *
+	 * [--refresh]
+	 * : Refresh site publish if link has expired.
+	 *
+	 * [--token=<token>]
+	 * : ngrok token.
+	 */
```
Comment (rahulsprajapati) [inline src/WordPress.php]: Examples? Need it for documentation.

File: src/WordPress.php [hunk 2]
```diff
@@ -1296,6 +1296,49 @@ public function reload( $args, $assoc_args, $whitelisted_containers = [], $reloa
...
+	public function publish( $args, $assoc_args ) {
+
+		parent::publish( $args, $assoc_args );
+
+		$disable = \EE\Utils\get_flag_value( $assoc_args, 'disable', false );
```
Comment (rahulsprajapati) [inline src/WordPress.php]: Please use nemespace and then it's a function. Don't use it directly. we are already refactoring this and that won't matter if we keep adding new code with same issue.

File: src/WordPress.php (hunk 1 above)
Comment (mrrobot47) [reply src/WordPress.php]: Added :+1:

File: src/WordPress.php (hunk 2 above)
Comment (mrrobot47) [reply src/WordPress.php]: Updated :+1:

**Pull request merged**